crate/cli/src/actions/findex/insert_or_delete.rs

```diff
@@ -1,19 +1,21 @@
+use super::parameters::FindexParameters;
+use crate::{
+    actions::findex::MAX_SEMAPHORES,
+    error::{result::CliResult, CliError},
+};
+use clap::Parser;
+use cosmian_findex::{Findex, IndexADT, Value};
+use cosmian_findex_client::FindexRestClient;
+use cosmian_findex_structs::{Keyword, KeywordToDataSetsMap, WORD_LENGTH};
```

Comment:
Do not add formatting changes.

Reply:
right

Reply:
Imho it doesn't really matters whether imports are sorted or not. I understand the want to enable automatic formatting though. Just keep in mind that formatting diffs bring increase the difficulty of a review. It is not always easy to see whether real changes lie among the formatting diffs. Maybe try delaying formatting diffs to dedicated PR when possible.